### PR TITLE
Enable prefer-const tslint flag

### PR DESCRIPTION
**Pull request by manishgarg1**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Button/BaseButton.styles.ts [hunk 1]
```diff
@@ -24,17 +24,17 @@ const iconStyle = {
 
 /**
  * Gets the base button styles. Note: because it is a base class to be used with the `mergeRules`
- * helper, it should have values for all class names in the interface. This let `mergeRules` optimize
+ * helper, it should have values for all class names in the interface. This const `mergeRules` optimize
```
Comment (cschleiden) [inline packages/office-ui-fabric-react/src/components/Button/BaseButton.styles.ts]: How did you make the changes? search/replace or using tslint autofix? We shouldn't change comments

Comment (manishgarg1) [reply packages/office-ui-fabric-react/src/components/Button/BaseButton.styles.ts]: Thanx for the review @cschleiden. This and a few others are already fixed.

File: packages/office-ui-fabric-react/tslint.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "no-any": false,
     "variable-name": false,
     "no-internal-module": false,
-    "prefer-const": false,
+    "prefer-const": true,
```
Comment (dzearing) [inline packages/office-ui-fabric-react/tslint.json]: This should just be deleted, and you should update the default oufr-tslint/tslint.json to be true.

Comment (manishgarg1) [reply packages/office-ui-fabric-react/tslint.json]: Got it. Let me fix that.

Comment (manishgarg1) [reply packages/office-ui-fabric-react/tslint.json]: done

---
In reply to: [168983208](https://github.com/OfficeDev/office-ui-fabric-react/pull/4018#discussion_r168983208) [](ancestors = 168983208)

**Comment by dzearing** [comment]
I have disabled the pr deploy workflow in vsts for now.
Requeuing the build.

**Pull request merged**